wolfssl/ssl.h

```diff
@@ -69,7 +70,6 @@
         #include <openssl/err.h>
         #include <openssl/ec.h>
         #include <openssl/hmac.h>
-        #include <openssl/bn.h>
```

Comment:
moving this and constantly include wolfssl/openssl/bn.h effects the opensslcoexist build